neqo-transport/src/connection.rs

```diff
@@ -411,6 +438,8 @@ pub struct Connection {
     /// During the handshake at the server, it also includes the randomized DCID pick by the client.
     valid_cids: Vec<ConnectionId>,
     retry_info: Option<RetryInfo>,
+    odcid: Option<ConnectionId>,
+    rscid: Option<ConnectionId>,
```

Comment:
It seems to me that you will have a small collection of these, which might be best put into a small struct.

I see only two here, but there are three to validate overall.

Reply:
the third goes in RetryInfo though...